src/main/kotlin/no/nav/syfo/Bootstrap.kt

```diff
@@ -590,6 +615,21 @@ suspend fun findNavkontorNr(
     return finnBehandlendeEnhetListeResponse?.behandlendeEnhetListe?.firstOrNull()?.enhetId ?: NAV_OPPFOLGING_UTLAND_KONTOR_NR
 }
 
+@KtorExperimentalAPI
```

Comment:
Kanskje vi kunne hatt en egen service for det som har med å finne riktig nav-kontor som fanger opp både denne, og metoden lenger opp?

Reply:
Jepp